requirements-dev.txt

```diff
@@ -13,7 +13,7 @@ pytest-cov==2.6.0
 requests-mock==1.5.2
 testfixtures==6.3.0
 
-git+https://github.com/alphagov/digitalmarketplace-test-utils.git@2.1.1#egg=digitalmarketplace-test-utils==2.1.1
+git+https://github.com/alphagov/digitalmarketplace-test-utils.git@ks-api-stubs-brief-response#egg=digitalmarketplace-test-utils==2.2.0
```

Comment:
dev commit :)

Reply:
It's so the CI passes while I'm waiting for the test utils PR to be merged 😸